docs/getting_started/tutorial.md

```diff
@@ -818,6 +818,8 @@ class BlogCategory(models.Model):
 
 ```{note}
 Note that we are using `panels` rather than `content_panels` here - since snippets generally have no need for fields such as slug or publish date, the editing interface for them is not split into separate 'content' / 'promote' / 'settings' tabs as standard, and so there is no need to distinguish between 'content panels' and 'promote panels'.
+
+Also note that the snippet is registered using `register_snippet` as a decorator for convenience. Later on, you might want to use it as a function in your `wagtail_hooks.py` as explained in [](wagtailsnippets_registering).
```

Comment:
I think this is the sort of non-essential background information that's best left out of a tutorial, as per https://diataxis.fr/tutorials/#ignore-options-and-alternatives.

Reply:
Thanks, I agree.